### DISCUSSION TITLE
一些面板问题疑问

### DISCUSSION THREAD

**Discussion by hostyun** [midoks/mdserver-web#623] [Q&A]
像是什么cssjs文件丢失，但是其实不是，我把webrtc启用就好了，难不成访问个面板还必须要把wertc启用？？？来获取别人的真实IP吗

其他面板都没出现过这种情况
![photo_2024-09-26_05-07-15](https://github.com/user-attachments/assets/b1ef1721-006a-4b19-95dd-b4042d9a2135)

**Comment by hostyun** [comment]
有时候强制刷新一下又好了，但是经常就出现这种情况。点什么都点不动，只要是弹出内的窗口。我只要把webrtc启用，就没出现过这样的问题。用其他面板没有发现有这种现象。

**Comment by midoks** [comment]
可能是终端的js（全局）影响了吧。
